### PR TITLE
First draft of README

### PR DESCRIPTION
**Pull request by nicolebussola**
- Add introduction
- add skeleton

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,81 @@
...
+* [Versioning](#versioning)
+* [Authors](#authors)
+* [Contact](#contact)
+* [License](#license)
+* [Roadmap](#roadmap)
+* [Contribution guidelines](#contribution-guidelines)
+* [Acknowledgements](#acknowledgements)
+
+## Motivation
+The histo-pathological analysis of tissue sections is the gold standard to assess the presence  and understand the nature of many complex diseases, such as tumors.
```
Comment (alessiamarcolini) [inline README.md:27]: extra whitespace after presence

Comment (alessiamarcolini) [reply README.md:27]: and ... presence of what?

File: README.md [hunk 2]
```diff
@@ -4,7 +4,81 @@
...
+* [License](#license)
+* [Roadmap](#roadmap)
+* [Contribution guidelines](#contribution-guidelines)
+* [Acknowledgements](#acknowledgements)
+
+## Motivation
+The histo-pathological analysis of tissue sections is the gold standard to assess the presence  and understand the nature of many complex diseases, such as tumors. 
+In daily practice, pathologists usually perform microscopy examination of tissue slides considering a limited number of regions and the clinical evaulation relies on several factors such as nuclei morphology, cell distribution, and color (staining): this process is time consuming, could lead to information loss, and suffers from inter-observer variability.
+
+The advent of digital pathology is changing the way patholgists work and collaborate, and has opened the way to a new era in computational pathology. In particular, histopathology is expected to be at the centre of the AI revolution in medicine [1], prevision supported by the increasing success of deep learning applications to digital pathology.
```
Comment (alessiamarcolini) [inline README.md:30]: centre -> center (let's stick with American english)

File: README.md [hunk 3]
```diff
@@ -4,7 +4,81 @@
...
+
+Whole Slide Images (WSIs), namely the translation of tissue slides from glass to digital format, are a great source of information from both a medical and a computational point of view. WSIs can be coloured with different staining technicques (e.g. H&E or IHC), and are usually very large in size (up to several GB per slide). Because of WSIs typical pyramidal structure, images can be retrieved at different magnification factors, providing a further layer of information beyond color.
```
Comment (alessiamarcolini) [inline README.md:32]: technicques -> techniques

File: README.md [hunk 4]
```diff
@@ -4,7 +4,81 @@
...
+
+However, processing WSIs is far from being trivial. First of all, WSIs can be stored in different proprietary formats, according to the scanner used to digitalized the slides, and a standard protocol is still missing. WSIs can also present artifacts, such as shadows, mold, or annotations (pen marks) that are not useful. Moreover, giving their dimensions, it is not possible to process a WSI all at once, or, for example, to feed a neural network: it is necessary to crop smaller regions of tissues (tiles), which in turns require a tissue detection step.
```
Comment (alessiamarcolini) [inline README.md:34]: digitalized -> digitalize

File: README.md [hunk 5]
```diff
@@ -4,7 +4,81 @@
...
+
+### Quickstart
+
+COMING SOON!
+
+## Versioning
+
+We use [PEP 440](https://www.python.org/dev/peps/pep-0440/) for versioning. 
+
+## Authors
```
Comment (alessiamarcolini) [inline README.md:63]: Maybe we could add a link to the github profiles

File: README.md [hunk 6]
```diff
@@ -4,7 +4,83 @@
...
+
+However, processing WSIs is far from being trivial. First of all, WSIs can be stored in different proprietary formats, according to the scanner used to digitalized the slides, and a standard protocol is still missing. WSIs can also present artifacts, such as shadows, mold, or annotations (pen marks) that are not useful. Moreover, giving their dimensions, it is not possible to process a WSI all at once, or, for example, to feed a neural network: it is necessary to crop smaller regions of tissues (tiles), which in turns require a tissue detection step.  
+
+The aim of this project is to provide a tool for WSI processing in a reproducible environment to support clinical and scientific research. Histolab is designed to handle WSIs, automatically detect the tissue, and retrieve informative tiles, and it can thus be integrated in a deep learning pipeline.
+
+See the detailed documentation [here]().
```
Comment (alessiamarcolini) [inline README.md:38]: also the link is missing, I know... but by default it redirects here https://github.com/MPBA/histolab/blob/write_readme which gives 404

Comment (nicolebussola) [reply README.md:38]: should I remove it?

Comment (alessiamarcolini) [reply README.md:38]: until we have the docs hosted somewhere, yes

File: README.md [hunk 7]
```diff
@@ -4,7 +4,81 @@
...
+
+## Versioning
+
+We use [PEP 440](https://www.python.org/dev/peps/pep-0440/) for versioning. 
+
+## Authors
+
+* **[Alessia Marcolini](https://github.com/alessiamarcolini)** 
+* **[Ernesto Arbitrio](https://github.com/ernestoarbitrio)**
+* **[Nicole Bussola](https://gitlab.fbk.eu/bussola)**
```
Comment (alessiamarcolini) [inline README.md:65]: why your GitLab account?

File: README.md [hunk 8]
```diff
@@ -4,7 +4,81 @@
...
+* [Authors](#authors)
+* [License](#license)
+* [Roadmap](#roadmap)
+* [Acknowledgements](#acknowledgements)
+* [References](#references)
+* [Contribution guidelines](#contribution-guidelines)
+
+
+## Motivation
+The histo-pathological analysis of tissue sections is the gold standard to assess the presence  of many complex diseases, such as tumors, and understand their nature.
```
Comment (alessiamarcolini) [inline README.md:27]: still extra whitespace after presence

File: README.md (hunk 7 above)
Comment (nicolebussola) [reply README.md:65]: cause my github is empty :)

**Pull request merged**